commitizen/cli.py

```diff
@@ -240,6 +240,12 @@
                         "help": "commit message that needs to be checked",
                         "exclusive_group": "group1",
                     },
+                    {
+                        "name": ["--reject-abort"],
+                        "action": "store_true",
+                        "default": False,
```

Comment:
If I'm not misunderstood, the original behavior is `True`, right? If that's the case, I'd prefer to use that as the default option, unless we're to bump our major version

Reply:
Sounds good 👍